WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -143,7 +143,7 @@ public void onBindViewHolder(GridViewHolder holder, int position) {
                 if (SiteUtils.isPhotonCapable(mSite)) {
                     thumbUrl = PhotonUtils.getPhotonImageUrl(media.getUrl(), mThumbWidth, mThumbHeight);
                 } else {
-                    thumbUrl = UrlUtils.removeQuery(media.getUrl()) + "?w=" + mThumbWidth + "&h=" + mThumbHeight;
+                    thumbUrl = UrlUtils.removeQuery(media.getUrl());
```

Comment:
I'm very tempted to use `media.getThumbnailUrl()` here instead, thumbnails are 150x150 by default in WordPress, that makes the media grid ugly but images load much faster (Average of 8kb images vs. 2Mb).